sentry_sdk/integrations/strawberry.py

```diff
@@ -322,29 +310,29 @@ def _sentry_patched_sync_view_handle_errors(self, errors, response_data):
 
     def _sentry_patched_handle_errors(self, errors, response_data):
         # type: (Any, List[GraphQLError], GraphQLHTTPResponse) -> None
-        hub = Hub.current
-        integration = hub.get_integration(StrawberryIntegration)
+        client = sentry_sdk.get_client()
+        integration = client.get_integration(StrawberryIntegration)
         if integration is None:
             return
 
         if not errors:
             return
 
-        with hub.configure_scope() as scope:
-            event_processor = _make_response_event_processor(response_data)
-            scope.add_event_processor(event_processor)
+        scope = Scope.get_isolation_scope()
```

Comment:
```suggestion
        scope = Scope.get_isolation_scope()
        scope.generate_propagation_context()
```
Same question as above :-)